.github/emulator/bigtable.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -eux
+
+docker pull -q gcr.io/google.com/cloudsdktool/cloud-sdk:316.0.0-emulators
+docker run \
+  -d \
+  -p 8086:8086 \
+  gcr.io/google.com/cloudsdktool/cloud-sdk:316.0.0-emulators gcloud beta emulators bigtable start --host-port=0.0.0.0:8086
```

Comment:
You know, I think we could make this generic and use just a single file for all these emulators.

Rename this to `start-emulator.sh`

```suggestion
  gcr.io/google.com/cloudsdktool/cloud-sdk:316.0.0-emulators gcloud beta emulators $1 start --host-port=0.0.0.0:8086
```

Then in the yaml file:

Replace
```
- run: ./.github/emulator/bigtable.sh
```

With
```
- run: ./.github/emulator/start-emulator.sh bigtable
```